docs/content/develop/run-tests.md

```diff
@@ -0,0 +1,344 @@
+    Replace `PROJECT_ID` with the ID of the Google Cloud project you are using for testing.
+
+1. Optional: Some tests may require additional variables to be set, such as:
+
+    ```
+    GOOGLE_ORG
+    GOOGLE_BILLING_ACCOUNT
+    ```
+
+## Run automated tests
+
+{{< tabs "version" >}}
+
+{{< tab "GA Provider" >}}
+
+1. Run unit tests and linters
+
+    ```bash
+    cd $GOPATH/src/github.com/hashicorp/terraform-provider-google
+    make test
+    make lint
+    ```
+
+
+1. Run acceptance tests for only modified resources. (Full test runs can take over 9 hours.) See [Go's documentation](https://pkg.go.dev/cmd/go#hdr-Testing_flags) for more information about `-run` and other flags.
+
+    ```bash
+    make testacc TEST=./google TESTARGS='-run=TestAccContainerNodePool'
+    ```
+
+
+
+1. Optional: Save verbose test output to a file for analysis.
+
+    ```bash
+    TF_LOG=TRACE make testacc TEST=./google TESTARGS='-run=TestAccContainerNodePool_basic' > output.log
+    ```
+
+1. Optional: Debug tests with [Delve](https://github.com/go-delve/delve). See [`dlv test` documentation](https://github.com/go-delve/delve/blob/master/Documentation/usage/dlv_test.md) for information about available flags.
+
+    ```bash
+    cd google
+    TF_ACC=1 dlv test -- --test.v --test.run TestAccComputeRegionBackendService_withCdnPolicy
+    ```
+
+{{< /tab >}}
+
+{{< tab "Beta Provider" >}}
+
+1. Run unit tests and linters
+
+    ```bash
+    cd $GOPATH/src/github.com/hashicorp/terraform-provider-google-beta
+    make test
+    make lint
+    ```
+
+
+1. Run acceptance tests for only modified resources. (Full test runs can take over 9 hours.) See [Go's documentation](https://pkg.go.dev/cmd/go#hdr-Testing_flags) for more information about `-run` and other flags.
+
+    ```bash
+    make testacc TEST=./google-beta TESTARGS='-run=TestAccContainerNodePool'
+    ```
+
+
+
+1. Optional: Save verbose test output to a file for analysis.
+
+    ```bash
+    TF_LOG=TRACE make testacc TEST=./google-beta TESTARGS='-run=TestAccContainerNodePool_basic' > output.log
+    ```
+
+1. Optional: Debug tests with [Delve](https://github.com/go-delve/delve). See [`dlv test` documentation](https://github.com/go-delve/delve/blob/master/Documentation/usage/dlv_test.md) for information about available flags.
+
+    ```bash
+    cd google-beta
+    TF_ACC=1 dlv test -- --test.v --test.run TestAccComputeRegionBackendService_withCdnPolicy
+    ```
+
+{{< /tab >}}
+
+{{< /tabs >}}
+
+## Test with different `terraform` versions
+
+Tests will use whatever version of the `terraform` binary is found on your `PATH`. If you are testing a change that you know only impacts certain `terraform` versions, follow these steps:
+
+1. Install [`tfenv`](https://github.com/tfutils/tfenv).
+
+1. Install the version of `terraform` you want to test.
+
+    ```bash
+    tfenv install VERSION
+    ```
+
+    Replace `VERSION` with the version you want to test.
+
+1. Run automated tests following the earlier section.
```

Comment:
please add a link to the earlier section here & in the next item

Reply:
I've removed the next item because all the tests are now under the automated tests section.